src/Socket.c

```diff
@@ -1276,14 +1281,14 @@ int Socket_new(const char* addr, size_t addr_len, int port, SOCKET* sock)
 	return codes from send, for testing only!
 */
 #if defined(SMALL_TCP_BUFFER_TESTING)
-        if (1)
-				{
-					int optsend = 100; //2 * 1440;
-					printf("Setting optsend to %d\n", optsend);
-					if (setsockopt(*sock, SOL_SOCKET, SO_SNDBUF, (void*)&optsend, sizeof(optsend)) != 0)
-						Log(LOG_ERROR, -1, "Could not set SO_SNDBUF for socket %d", *sock);
-				}
-	#endif
+			if (1)
+			{
+				int optsend = 100; //2 * 1440;
+				printf("Setting optsend to %d\n", optsend);
+				if (setsockopt(*sock, SOL_SOCKET, SO_SNDBUF, (void*)&optsend, sizeof(optsend)) != 0)
+					Log(LOG_ERROR, -1, "Could not set SO_SNDBUF for socket %d", *sock);
+			}
+#endif
```

Comment:
you are not changing any code -> ident issue. The diff should be empty.

Reply:
I disagree in the sense that the previous indenting was incorrect and misleading. Admittedly, this has nothing to do with the rest of the PR; it's a holdover from my initial attempt to modify `Socket_new()` for UNIX sockets, before deciding that adding a separate `Socket_unix_new()` would be more clear.

I'd prefer to keep it.